src/python/turicreate/toolkits/style_transfer/_tf_model_architecture.py

```diff
@@ -24,32 +24,37 @@ def define_tensorflow_variables(net_params, trainable=True):
     Parameters
     ----------
     trainable: boolean
-        If `true` the network updates the convolutional layers as well as the
-        instance norm layers of the network. If `false` only the instance norm
-        layers of the network are updated.
+        If `True` the transformer network updates the convolutional layers as
+        well as the instance norm layers of the network. If `False` only the
+        instance norm layers of the network are updated.
+
+        Note the VGG network's parameters aren't updated
     Returns
     -------
     out: dict
         The TF Variable dictionary.
     """
     tensorflow_variables = dict()
     for key in net_params.keys():
-        if "weight" in key:
-            if "conv" in key:
+        if 'weight' in key:
```

Comment:
Why are we going back to single quotes here? Are not using the new formatter/linter?